rust/agama-lib/src/storage/client.rs

```diff
@@ -16,18 +21,112 @@ pub struct StorageDevice {
     description: String,
 }
 
+/// Represent single change action done to storage
+#[derive(Debug, Clone, Serialize, utoipa::ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct Action {
+    device: String,
+    text: String,
+    subvol: bool,
+    delete: bool,
+}
+
+/// Represents value for target key of Volume
+#[derive(Debug, Clone, Serialize, Deserialize, utoipa::ToSchema)]
+#[serde(rename_all = "snake_case")]
```

Comment:
I guess you are using `snake_case` because it comes like that from storage-ng, right? I would add a note there.

Reply:
yes, I try to use same values...in fact currently on D-Bus it is just string...so I think it would be nicer to have enum from it.